### PR TITLE
Show match notification dialog after creating target

### PR DESCRIPTION
**Pull request by marcossalto**
Show match notification dialog after creating target

Screenshot_1615495735

### CODE REVIEW

**Review by amaury901130 — APPROVED** [review]
💪

File: app/src/main/java/com/marcossalto/targetmvd/network/models/Target.kt [hunk 1]
```diff
@@ -31,3 +42,10 @@ fun Target.toTargetModel(topic: TopicModel): TargetModel {
         id = id
     )
 }
+
+fun Conversation.toConversationModel(): ConversationModel {
+    return ConversationModel(
+        id=id,
```
Comment (ximenaperez) [inline app/src/main/java/com/marcossalto/targetmvd/network/models/Target.kt]: add spaces between the equal sign

File: app/src/main/java/com/marcossalto/targetmvd/network/models/User.kt [hunk 2]
```diff
@@ -62,3 +72,28 @@ data class UserSignUp(
...
+    @Json(name = "last_name") var lastName: String? = "",
+    @Json(name = "full_name") var fullName: String? = "",
+    @Json(name = "username") val username: String,
+    @Json(name = "gender") var gender: String,
+    @Json(name = "avatar") val avatar: AvatarMatch
+)
+
+fun MatchedUser.toMatchedUserModel(): MatchedUserModel {
+    return MatchedUserModel(
+        id=id,
```
Comment (ximenaperez) [inline app/src/main/java/com/marcossalto/targetmvd/network/models/User.kt]: same here

File: app/src/main/java/com/marcossalto/targetmvd/ui/target/TargetView.kt [hunk 3]
```diff
@@ -119,6 +121,36 @@ class TargetView(
...
+        Glide.with(bindingRoot.context)
+            .load(R.mipmap.avatar)
+            .circleCrop()
+            .into(dialogView.avatar)
+
+        matchedUserModel.fullName.also { dialogView.match_name_text_view.text = it }
+
+        alertDialog.show()
+
+        dialogView.cool_start_chatting_button.setOnClickListener{
```
Comment (ximenaperez) [inline app/src/main/java/com/marcossalto/targetmvd/ui/target/TargetView.kt]: add space before the opening bracket

File: app/src/main/java/com/marcossalto/targetmvd/ui/target/TargetView.kt [hunk 4]
```diff
@@ -119,6 +121,36 @@ class TargetView(
...
+            alertDialog.dismiss()
+            successCreatingTarget()
+            startChat()
+        }
+        dialogView.skip_button.setOnClickListener{
```
Comment (ximenaperez) [inline app/src/main/java/com/marcossalto/targetmvd/ui/target/TargetView.kt]: same here

File: app/src/main/java/com/marcossalto/targetmvd/ui/target/TargetView.kt [hunk 5]
```diff
@@ -139,6 +171,10 @@ class TargetView(
         })
     }
 
+    private fun observeMatchConversation(){
```
Comment (ximenaperez) [inline app/src/main/java/com/marcossalto/targetmvd/ui/target/TargetView.kt]: and here

File: app/src/main/java/com/marcossalto/targetmvd/network/models/Target.kt (hunk 1 above)
Comment (marcossalto) [reply app/src/main/java/com/marcossalto/targetmvd/network/models/Target.kt]: Thanks  for your suggestion

File: app/src/main/java/com/marcossalto/targetmvd/network/models/User.kt (hunk 2 above)
Comment (marcossalto) [reply app/src/main/java/com/marcossalto/targetmvd/network/models/User.kt]: Ok

File: app/src/main/java/com/marcossalto/targetmvd/ui/target/TargetView.kt (hunk 3 above)
Comment (marcossalto) [reply app/src/main/java/com/marcossalto/targetmvd/ui/target/TargetView.kt]: Thanks for your suggestion

File: app/src/main/java/com/marcossalto/targetmvd/ui/target/TargetView.kt (hunk 4 above)
Comment (marcossalto) [reply app/src/main/java/com/marcossalto/targetmvd/ui/target/TargetView.kt]: Thanks for your suggestion

File: app/src/main/java/com/marcossalto/targetmvd/ui/target/TargetView.kt (hunk 5 above)
Comment (marcossalto) [reply app/src/main/java/com/marcossalto/targetmvd/ui/target/TargetView.kt]: Thanks for your suggestion

**Review by ximenaperez — APPROVED** [review]

**Pull request merged**